### DISCUSSION TITLE
Best approach to level collision: 16x16 tiles, or 256x256 chunks?

### DISCUSSION THREAD

**Discussion by JimmyThrush** [Techokami/SonicWorldsNext#146] [General]
My approach to laying out levels so far has been to apply the graphics with 256x256 tiles on a physics-less tilemap, and then to apply collision with a separate invisible tilemap comprised of 16x16 tiles. From my limited understanding of the original Genesis games' code, that's how they handled it as well. 

However, I noticed that there's been some experimentation with 256x256 chunk-based collision in SWN (with "Chunk Zone" and its associated tileset), though there's been little discussion of this subject on this repo from what I've seen. What would you say are the pros and cons of each approach?

**Comment by Renhoex** [comment]
Chunk zone became deprecated in the move up to Godot 4 since you can just set a collision in a 256x256 tileset without much lag.

The thing is there's really not one way to do any of this, both options are equally as valid, it's just what you're more comfortable with.
